src/components/Voting/Referendum/Items/ProposalItem/index.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Cant we remove the import?

Reply:
Probably not, in next lines it's used

`  } else {
    // this sux but we have no other way until we refactor it to NOT use toJSON/toHuman
    hash = blake2AsHex(hexToU8a(boundedCall.Inline));
  }`

Reply:
Then why `eslint-disable-next-line` is needed?

Reply:
Ahh, I see. Cause we don't have it in package.json, but use it directly. Can we add it to package.json instead of sprinkling eslint disables?

Reply:
yea, i will ad it